lib/rules/no-continue.js

```diff
@@ -11,6 +11,8 @@
 
 module.exports = {
     meta: {
+        type: "style",
```

Comment:
Since fixing errors reported by this rule would require refactoring the code, should this be a "suggestion"?